pkg/controller/logstorage/logstorage_controller_test.go

```diff
@@ -1315,6 +1320,7 @@ func setUpLogStorageComponents(cli client.Client, ctx context.Context, storageCl
 		nil, nil, "cluster.local", nil, render.ElasticsearchLicenseTypeBasic)
 
 	createObj, _ := component.Objects()
+	createObj = append(createObj, ls)
```

Comment:
I think instead of adding this here, you should add an `Expect(cli.Create(ctx, ls)).ShouldNot(HaveOccurred())` before the `render.LogStorage`.

Reply:
Change applied:
```diff
+       Expect(cli.Create(ctx, ls)).ShouldNot(HaveOccurred())
        createObj, _ := component.Objects()
-       createObj = append(createObj, ls)
```